logstash-filter-grok.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-filter-grok'
-  s.version         = '3.2.4'
+  s.version         = '3.2.5'
```

Comment:
This should be a minor bump due to the feature addition and default behavior change.